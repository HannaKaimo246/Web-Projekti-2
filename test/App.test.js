import React from 'react';
import renderer from 'react-test-renderer';
import {Card} from 'react-bootstrap';
const handleReset = require('../src/components/User/Register');



test('is input in right form', () => {
  const component = renderer.create(
      <Card page="http://www.facebook.com">Facebook</Card>,
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});