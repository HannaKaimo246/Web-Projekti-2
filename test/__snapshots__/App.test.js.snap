// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`is input in right form 1`] = `
<div
  className="card"
  page="http://www.facebook.com"
>
  Facebook
</div>
`;
